teamC/dayoung/services/userService.js

```diff
@@ -0,0 +1,21 @@
+const userDao = require('../models/userDao')
+
+const signUp = async (name, email, password, profileImage) => {
+    // password validation using REGEX
+    const pwValidation = new RegExp(
+      '^(?=.*[A-Za-z])(?=.*[0-9])(?=.*[!@#$%^&*])(?=.{8,20})'
+    );
+    
+    if (!pwValidation.test(password)) {
+      const err = new Error('PASSWORD_IS_NOT_VALID');
+      err.statusCode = 409;
```

Comment:
상태코드 409는 PUT method를 사용할 때 데이터와 충돌 할 경우 주로 사용됩니다.
front에서 잘 못된 입력 값을 준 것이므로 400으로도 충분할 것 같습니다.
```suggestion
      err.statusCode = 400;
```

Reply:
수정했습니다.